### ISSUE TITLE
White line on the left - part of menu of Fully Kiosk Browser showing

### ISSUE THREAD

**Issue by xteaun** [j-a-n/lovelace-wallpanel#92]
On my Amazon fire hd 10, there remains a white line visible on the left when using Fully Kiosk Browser. This only happens when using wallpanel, not with other websites. So it seems to be a wallpanel-issue, not a Fully Kiosk Browser-issue. How do I remove the white line?

Thanks!
Christian

**Comment by jon6fingrs** [comment]
I just installed lovelace-wallpanel for use with a fire hd 8 wallpanel since the kiosk-mode extension seems to have stopped working, and have also noticed this white line.

I am using the following config:
```

wallpanel:
  enabled: true
  hide_toolbar: true
  hide_sidebar: true
  fullscreen: true
  idle_time: 0
```

Thanks!

**Comment by j-a-n** [comment]
Could somebody please post a screenshot?

**Comment by rehakk** [comment]
Repository 'j-a-n/lovelace-wallpanel' exists in the store.

What's next?

**Comment by j-a-n** [comment]
The issue should be fixed with version 4.9.1

**Comment by jon6fingrs** [comment]
Wow thanks!
